canonical-websites/jp.ubuntu.com
# Simple cloudinary test

## Done

- Simple auto improvment of images with cloudinary
- Moved js files to a new TEXT_ASSETS_SERVER_URL variable

## QA

1. Download this branch
2. Click through all the pages and see there are no errors in the network tab
